app/lib/frontend/templates/views/pkg/admin_page.dart

```diff
@@ -155,5 +159,64 @@ d.Node packageAdminPageNode({
         checked: package.isUnlisted,
       ),
     ],
+    if (requestContext.showPackageRetraction) ...[
+      d.h2(text: 'Package Version Retraction'),
+      d.div(children: [
+        d.markdown(
+            'You can retract a package version up to 7 days after publication.'),
+        d.markdown(
+            'This will not remove the package version, but warn developers using'
+            ' it and stop new applications from taking dependency on it without a dependency override.'),
+        d.markdown(
+            'You can restore a retracted package version if the version was retracted within the last 7 days.'),
```

Comment:
Can't this be merged to a single markdown block...

with line breaks or something.. Just seems like it would be easier to write as:
```dart
      d.markdown('''
....

....
      ''')
```

Or
```
      d.markdown([
         '...',
         '',
         '....',
      ].join('\n'))
```

Reply:
The bottom approach does not seem to work, the top one does. However I think it is less readable, so unless you feel strongly about it, I'll leave it as is.